sirbot/pythondev/slack/do.py

```diff
@@ -0,0 +1,17 @@
+DIGITAL_OCEAN_REFFERRAL_URL = 'https://m.do.co/c/457f0988c477'
+DIGITAL_OCEAN_URL = 'https://digitalocean.com'
+
+DIGITAL_OCEAN_TEXT = '''Here at Python Developers we host our website and
+Slack bot on <{}|Digital Ocean>. If you are planning on using Digital Ocean,
+please use our <{}|referral code>. You get 10 USD and help support the
+ community!'''.format(DIGITAL_OCEAN_URL, DIGITAL_OCEAN_REFFERRAL_URL)
```

Comment:
Maybe be more explicit about how it support the community ? Hosting fees for the website and sirbot